### PR TITLE
fix(sla deviation): change deviation percent

### PR DESCRIPTION
**Pull request by juliayakovlev**
By deviation percent we decide if the test result is as expected or not.
In ideal expected ratio between two users is 5.0.
It was decided to allow 10% deviation (from 4.1 to 5.0 ratio)
Based on reality change it to 32% (from 3.4 to 5.0 ratio)

Accroding to Roy's request:
http://13.48.103.68/test/71402aa7-051b-4803-a6b4-384529680fb7/runs?additionalRuns%5B%5D=1adf34d1-15cf-4973-80ce-9de130be0b09

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Review by fruch — APPROVED** [review]
LGTM

**Comment by fruch** [comment]
@roydahan waiting for you approval for backporting this

**Review by eliransin — APPROVED** [review]
LGTM. I hope that we will be able to reach closer to our goal but it is best to stick to what reality dictates :slightly_smiling_face:

**Comment by roydahan** [comment]
TBH, I don’t think this is correct.
I think that the expected ratio should be 3.5 (and 10-15% deviation).
IIRC, these are the numbers we’re getting.
Also, we need to tell @eliransin that we’re setting it as such.

**Comment by fruch** [comment]
> LGTM. I hope that we will be able to reach closer to our goal but it is best to stick to what reality dictates

this is what @eliransin commented, still doesn't answer if to backport or not

**Pull request merged**
